package.json

```diff
@@ -31,27 +31,25 @@
     "productName": "Tezori3",
     "files": [
       "build/**/*",
-      "node_modules/**/*",
-      "main.js"
+      "electron/main.ts",
+      "package.json"
     ],
-    "directories": {
-      "buildResources": "public"
-    },
     "linux": {
       "target": "deb"
-    }
+    },
+    "extends": null
   },
-  "main": "electron-main.js",
+  "main": "electron/main.ts",
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test --passWithNoTests",
-    "eject": "react-scripts eject",
+    "start": "react-app-rewired start",
+    "build": "react-app-rewired build",
+    "test": "react-app-rewired test --passWithNoTests",
+    "eject": "react-app-rewired eject",
```

Comment:
Unfortunately, using `react-app-rewired` does not fix the Webpack v5 issue with the Ledger and Beacon libraries. Unless you can make it work, I suggest reverting this change.